transcriptclean.wdl

```diff
@@ -0,0 +1,199 @@
+version 1.0
+
+# Copyright (c) 2019 Sequencing Analysis Support Core - Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task transcriptclean {
+    input {
+        File SAMfile
+        File referenceFile
+        String outputPrefix
+        String outputDirPath
+        File? spliceJnsFile
+        File? variantsFile
+        Int? maxLenIndel
+        Int? maxSJoffset
+        Boolean? correctMismatches
+        Boolean? correctIndels
+        Boolean? correctSJs
+        Boolean? dryRun
+        Boolean? primaryOnly
+        Int cores = 1
+        Int memory = 25
+        String dockerImage = "biocontainers/transcriptclean:v1.0.7_cv1"
+    }
+
+    command {
+        set -e pipefail
+        mkdir -p ~{outputDirPath}
+        TranscriptClean \
+            ~{"-s " + SAMfile} \
```

Comment:
Why is this indented? It looks a little strange that way.

Reply:
The arguments I indented are additions to the TranscriptClean call, but not to the other bash calls (like mkdir) in the command section. So I thought the indents would make that separation more transparent.
I have removed the extra indents from the command sections.